.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,49 @@
+name: Go Benchmark Test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-*
+  push:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  check-changes:
+    name: Check whether tests need to be run based on diff
+    runs-on: [ ubuntu-latest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: antrea-io/antrea/ci/gh-actions/has-changes@main
+        id: check_diff
+        with:
+          args: pkg/*
+    outputs:
+      has_changes: ${{ steps.check_diff.outputs.has_changes }}
+
+  go-benchmark-checks:
+    needs: check-changes
+    if: ${{ needs.check-changes.outputs.has_changes == 'yes' || github.event_name == 'push' }}
+    name: GoBenchmark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Install Benchci
+        run: curl -sfL https://raw.githubusercontent.com/antrea-io/benchci/main/install.sh | sudo sh -s -- -b /usr/local/bin
+
+      - name: Run Benchmark
+        run: benchci -config test/performance/benchmark.yml
```

Comment:
@wenqiq running as a post-merge just means that we don't run it for Pull Requests a a required check. Instead we only run the workflow when the `main` branch is updated (push event).

Reply:
Thanks. I misunderstood. Then it will work well just change as fellows?
```
on:
  pull_request:
    types: [closed]
    branches:
      - main
  push:
    branches:
      - main
```

Reply:
Updated. @antoninbas @tnqn Do you have any more comments? Also updated some code in https://github.com/antrea-io/benchci/pull/2

Reply:
just:
```
on:
  push:
    branches:
      - main
```

Reply:
Done.